Informal List of Projects

```diff
@@ -0,0 +1,66 @@
+###Code Repository
+A kind of Gitbuh but decentralised. Rewards based on commits.
+
+###Security Cameras 
+Use case: you can install security cameras in a kindergarten but no-one can access the footage unless 51% of parents give the consent to do it. It would solve the privacy vs security issue.
+
+###Plugin as Coinpayments
+But open, as with myetherwallet, you can add custom token just defining name, address and decimals
+
+###Call-Center / Customer-Care on demand
+Uber economy; Decentralised customer care on demand.
+It would be also possible to create services for marketplaces (i.e. themeforest) where support is based on some reward system
+
+###Products Warranty / delivery tracking / dropshipping infrastructure
+bill/warranty/tracking on the blockchain (QRcode)
+You could also make automated order  fulfillment with smart-contracts for dropshipping services
+
+###Turnover
+Shifts system. I.e. a Netflix licens give me the right to use up to 4 devices but we could pay it in 20 people and use it according to a shared calendar.
+
+###Easy Mobile Investments
+Every day we send users a push notification e they can decide whether to invest 10, 20, 50 o 100 $ in just a click.
```

Comment:
This would be an ideal way to open the hedge fund to the public.

Reply:
I my idea only the admin has the password, the other ones can log-in using some kind of authenticathor widget